src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkVideoPlayer.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from "react"
+import { Box, ResponsiveBox } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkVideoPlayer_artwork } from "v2/__generated__/ArtworkVideoPlayer_artwork.graphql"
+
+interface ArtworkVideoPlayerProps {
+  artwork: ArtworkVideoPlayer_artwork
+  small?: boolean
```

Comment:
What does this do? I'm not sure why we'd need to differentiate on size

Reply:
unsets minHeight. looking at some of the changes you suggested, and will see if it remains necessary.

Reply:
A general rule of thumb: if you're setting an arbitrary height somewhere, there's an easier way.